src/main/java/com/es/phoneshop/utils/ReferenceTool.java

```diff
@@ -0,0 +1,18 @@
+package com.es.phoneshop.utils;
+
+public class ReferenceTool<T> {
```

Comment:
what for?

Reply:
this is for the counter of non-updated products. Because i needed to use reference-typed object (returning of the int counter is not a good variant) that will save counter from method to method. I used List<Integer>, but referenced integer i suppose is better